Feature/versions (#107)
infinum/eightshift-docs: website/docs/welcome.md

- **Cache busting** - Each time you build assets, we generate a new URL for each asset file. This means you'll never see the old cached version while developing or after pushing your code to production.

> To use it, you need to understand what makes this boilerplate tick. In theory, this is simple; you have two parts.

Reviewer: ```suggestion
To use it, you need to understand what makes this boilerplate tick. It consists of two parts:
```
